### PR TITLE
test: add e2e test for multi input and object marker 

### PR DESCRIPTION
**Pull request by nikvalor**
#### Please provide a brief summary of this pull request.
Added e2e test for multi input and object marker (platform and core)
#### Please check whether the PR fulfills the following requirements

- [ ] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [ ] tests for the changes that have been done
- [ ] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist

Documentation checklist:
- [ ] update `README.md`
- [ ] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [ ] Documentation Examples
- [ ] Stackblitz works for all examples

### CODE REVIEW

File: e2e/wdio/core/tests/object-marker.e2e-spec.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    afterEach(() => {
+        refreshPage();
+    });
+    it('Verify each marker is clickable', () => {
+        const arr = getElementArrayLength(objectMarkerPage.marker);
+        for (let i = 0; i < arr; i++) {
+            scrollIntoView(objectMarkerPage.marker, i);
+            click(objectMarkerPage.marker, i);
+        }
+    });
```
Comment (Seamoo13) [inline e2e/wdio/core/tests/object-marker.e2e-spec.ts:31]: Please add an empty line between the statements, for readability

File: e2e/wdio/platform/pages/multi-input.po.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import { BaseComponentPo } from './base-component.po';
+import { click, pause, scrollIntoView, sendKeys, waitForElDisplayed, waitForPresent } from '../../driver/wdio';
+
+export class MultiInputPo extends BaseComponentPo {
+    private url = '/multi-input';
+    root = '#page-content';
+    expandedDropdown = '.fd-list'
+    activeDropDownButtons = '[ng-reflect-disabled="false"] button[ng-reflect-glyph="value-help"]'
```
Comment (Seamoo13) [inline e2e/wdio/platform/pages/multi-input.po.ts:8]: for one you use 'Drop**d**own' and another 'Drop**D**own', let's pick 1 variant. I think 'Dropdown' is ok

File: e2e/wdio/platform/tests/multi-input.e2e-spec.ts [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+            for (let i = 0; i < activeButtonsQuantity; i++) {
+                if (i !== activeButtonsQuantity - 2) {
+                    multiInputPage.expandDropdown(multiInputPage.activeDropDownButtons, i);
+                    const optionsArr = getAttributeByNameArr(multiInputPage.options, 'ng-reflect-title');
+                    multiInputPage.selectOption(optionsArr[0]);
+                    expect(getText(multiInputPage.filledInput, i)).toContain(optionsArr[0]);
+                    expect(getText(multiInputPage.filledInput, i).split('\n')[0]).toBe(optionsArr[0]);
+                    click(multiInputPage.selectedToken);
+                    sendKeys(['Backspace']);
+                    sendKeys(['Backspace']);
```
Comment (Seamoo13) [inline e2e/wdio/platform/tests/multi-input.e2e-spec.ts:151]: should be able to do: sendKeys(['Backspace', 'Backspace']);

File: e2e/wdio/platform/tests/multi-input.e2e-spec.ts [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+                } else {
+                    multiInputPage.expandDropdown(multiInputPage.activeDropDownButtons, i);
+                    const optionsArr = getAttributeByNameArr(multiInputPage.options, 'ng-reflect-title');
+                    multiInputPage.selectOption(optionsArr[0]);
+                    click(multiInputPage.approveButton);
+                    expect(getText(multiInputPage.filledInput, i)).toContain(optionsArr[0]);
+                    expect(getText(multiInputPage.filledInput, i).split('\n')[0]).toBe(optionsArr[0]);
+                    click(multiInputPage.selectedToken);
+                    sendKeys(['Backspace']);
+                    sendKeys(['Backspace']);
```
Comment (Seamoo13) [inline e2e/wdio/platform/tests/multi-input.e2e-spec.ts:162]: same as above

**Review by Seamoo13 — CHANGES_REQUESTED** [review]

**Comment by Betrozov** [comment]
Your PR has 149 commits.
I think there should be less.
Could you fix it, please?

File: e2e/wdio/platform/tests/multi-input.e2e-spec.ts [hunk 5]
```diff
@@ -136,30 +136,28 @@ describe('Multi input test suite', function() {
         }
     });
 
-    it('Verify user can delete the token using backspace and delete key', () => {
+    fit('Verify user can delete the token using backspace and delete key', () => {
```
Comment (Seamoo13) [inline e2e/wdio/platform/tests/multi-input.e2e-spec.ts:139]: it

**Review by Seamoo13 — APPROVED** [review]

File: e2e/wdio/core/tests/object-marker.e2e-spec.ts [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+    iconStatusesList,
+    textDecorationAttribute,
+    textDecorationValues
+} from '../fixtures/appData/object-marker-content';
+describe('Object marker test suite', function() {
+    const objectMarkerPage = new ObjectMarkerPo();
+
+    beforeAll(() => {
+        objectMarkerPage.open();
+        pause(16000);
```
Comment (valorkin) [inline e2e/wdio/core/tests/object-marker.e2e-spec.ts:21]: replace with wait as we discussed

File: e2e/wdio/platform/tests/multi-input.e2e-spec.ts [hunk 7]
```diff
@@ -0,0 +1,174 @@
...
+} from '../../driver/wdio';
+import {placeholderValue} from '../fixtures/appData/file-uploader.page-content';
+import { MultiInputPo } from '../pages/multi-input.po';
+
+describe('Multi input test suite', function() {
+    const multiInputPage: MultiInputPo = new MultiInputPo();
+
+    beforeAll(() => {
+        multiInputPage.open();
+        pause(16000);
```
Comment (valorkin) [inline e2e/wdio/platform/tests/multi-input.e2e-spec.ts:19]: same

**Review by valorkin — CHANGES_REQUESTED** [review]
remove usage of pause

**Comment by nikvalor** [comment]
Reopened PR from another branch [4308](https://github.com/SAP/fundamental-ngx/pull/4308)

**Pull request closed without merging**